inloop/statistics/truncators.py

```diff
@@ -0,0 +1,19 @@
+import django.db.models.functions as functions
```

Comment:
This file can also be merged with `views.py`, if these are just utils for the view classes.

Reply:
Ok.